### ISSUE TITLE
Missing ACL_POOL in XSD schema

### ISSUE THREAD

**Issue by feldsam** [OpenNebula/one#4076]
**Description**
ACL Pool is defined, but not included in XSD index. There is also wrong definition on most field, which are integers but in reality are hexadecimals

**Details**
 - Affected Component: Core
 - Hypervisor: All
 - Version: All

## Progress Status
- [x] Branch created 
- [x] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by rsmontero** [comment]
Also needs xsd tests

**Comment by feldsam** [comment]
point me to dir where tests lives

**Comment by rsmontero** [comment]
NP it was more a note for us when reviewing, and merging. THANKS for your contributions and feed back!

**Comment by xorel** [comment]
tests (xsd + pyone) https://github.com/OpenNebula/development/pull/812

**Comment by feldsam** [comment]
@rsmontero hello, is this possible to cherry-pick also to 5.10? Because the final version `Use xs:string for ACLs` is only in the master branch.

**Comment by rsmontero** [comment]
AFAIK I merged both branches (master is actually a cherry pick from one-5.10):

one-5.10:
```
commit b19511f4b9208253f9e1f40da165905b9285858f
Author: Kristian Feldsam <feldsam@gmail.com>
Date:   Thu Apr 16 15:37:28 2020 +0200

    B #4148: Added missing showback xsd (#4149)
    
    Signed-off-by: Kristian Feldsam <feldsam@gmail.com>
```

master:
```
commit 3d29bb672fa6407b3a5e0cbb71235a99dc449e9f
Author: Kristian Feldsam <feldsam@gmail.com>
Date:   Thu Apr 16 15:37:28 2020 +0200

    B #4148: Added missing showback xsd (#4149)
    
    Signed-off-by: Kristian Feldsam <feldsam@gmail.com>
    (cherry picked from commit b19511f4b9208253f9e1f40da165905b9285858f)
```

**Comment by rsmontero** [comment]
Maybe, are you referring to a different commit?

**Comment by feldsam** [comment]
yes, this was merged, but after while reverted in this PR #4095 where is a discussion about it and finally, we decided to use `xs:string` in PR #4117

**Comment by rsmontero** [comment]
Merged it to one-5.10:

```
commit e590f1a055e21240f37c8fffe5f5f081df45908b (HEAD -> one-5.10, origin/one-5.10)
Author: Jan Orel <jorel@opennebula.systems>
Date:   Tue Jan 28 12:40:50 2020 +0100

    B #4076: Use xs:string for ACLs (#4117)
    
    hexBinary fails due to non-even length
    integer fails obviously when containig a-f chars
    
    (cherry picked from commit fe63e48eda8a095bcc61beee7f65cb691afa76cc)
```
